patched-vscode/extensions/code-editor-idle-extension/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "code-editor-idle-extension",
```

Comment:
would suggest prefixing with sagemaker since 1/ it's added by a sagemaker patch and 2/ it's only relevant when running app on SageMaker   
code-editor is not needed - `sagemaker-idle-extension`